packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -183,7 +183,7 @@ Documentation for %{pkg_name}.
 gem unpack %{SOURCE0}
 %{?scl:EOF}
 
-%setup -q -D -T -n  %{gem_name}-%{version}
+%setup -q -D -T -n  %{gem_name}-%{version}%{?prever}
```

Comment:
Sorry, this isn't actually needed, I was looking at this all wrong. Basically, let's mimic this https://github.com/theforeman/foreman-packaging/commit/5f45c183be9b087ce87f24bc4b87d86120d587b8#diff-a8930fa5b28236b4e1d2d7dc0775f97b